.yamato/config/_abv.metafile

```diff
@@ -5,84 +5,119 @@ abv: # specify the editor versions for which to create the expression trigger fo
     - name: HDRP
     - name: VFX_HDRP
     - name: VFX_URP
+  build_configs:
+    - name: mono_2
+      scripting_backend: mono
+      api_level: NET_2_0
+    - name: il2cpp_4
+      scripting_backend: IL2CPP
+      api_level: NET_4_6
+  color_spaces:
+    - Linear
 nightly:
   extra_dependencies: # additional project jobs to run nightly (in addition to ABV)
     - project: Smoke_Test
       platform: Win
       api: DX11
+      build_config: mono_2
+      color_space: Linear
       test_platforms:
         - playmode
         - editmode
         - Standalone
     - project: Universal
       platform: Android
       api: OpenGLES3
+      build_config: il2cpp_4
+      color_space: Linear
       test_platforms:
         - Standalone
     - project: Universal
       platform: Android
       api: Vulkan
+      build_config: il2cpp_4
+      color_space: Linear
       test_platforms:
         - Standalone
     - project: Universal
       platform: iPhone
       api: Metal
+      build_config: mono_2
+      color_space: Linear
       test_platforms:
         - Standalone
     - project: Universal
       platform: Win
       api: Vulkan
+      build_config: mono_2
+      color_space: Linear
       test_platforms:
         - playmode
-        - playmode_XR
     - project: Universal
       platform: Win
       api: DX12
+      build_config: mono_2
+      color_space: Linear
       test_platforms:
         - Standalone
         - editmode
         - playmode
-        # - playmode_XR # uncomment once bug fixed on XR side
```

Comment:
Related to a previous comment. so I guess this is intentional. Is the XR issue related to perf testing? iirc it works correctly currently.

Reply:
oh its commented out on master though,